### PR TITLE
Add E2E job which build and deploy operators

### PR DESCRIPTION
**Pull request by pojadhav**

### CODE REVIEW

**Comment by sandeepyadav93** [comment]
We need to set placement-operator image as public on quay.rdoproject registry(this is our test registry)

**Comment by pojadhav** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-placement-operator-master-placement-operator-build-deploy

**Comment by pojadhav** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-placement-operator-master-placement-operator-build-deploy

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-placement-operator-master-placement-operator-build-deploy

**Comment by Sandeepyadav93** [comment]
Last failure was on https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/36540/rehearse-36540-pull-ci-openstack-k8s-operators-placement-operator-master-placement-operator-build-deploy/1628430935999385600#1:build-log.txt%3A216

"Egress is over the account limit." - this got fixed yesterday

**Comment by Sandeepyadav93** [comment]
/ph-rehearse


rerunning to confirm if issue is consistent

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36540/rehearse-36540-pull-ci-openstack-k8s-operators-placement-operator-master-placement-operator-build-deploy/1628646369558269952/artifacts/placement-operator-build-deploy/openstack-k8s-operators-build/build-log.txt

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-placement-operator-master-placement-operator-build-deploy

**Comment by gibizer** [comment]
I manually bumped placement-operator version in openstack-operator to avoid the version conflict https://github.com/openstack-k8s-operators/openstack-operator/pull/209 hope after that lands we can make this pass

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-placement-operator-master-placement-operator-build-deploy

Rerunning to have fresh results

**Comment by Sandeepyadav93** [comment]
Looks good to me.

File: ci-operator/config/openstack-k8s-operators/placement-operator/openstack-k8s-operators-placement-operator-master.yaml [hunk 1]
```diff
@@ -48,6 +69,17 @@ tests:
...
     from: src
+- as: placement-operator-build-deploy
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  optional: true
```
Comment (gibizer) [inline ci-operator/config/openstack-k8s-operators/placement-operator/openstack-k8s-operators-placement-operator-master.yaml:80]: OK, so this is not voting. Cool.

**Review by gibizer — APPROVED** [review]
Looks good. Thanks for making this happen. 

We are aware of the fact that in case of a placement-operator change requires openstack-operator adaptation then this job will fail and we need to carefully ignore such result.

**Pull request merged**